runbot_travis2docker/models/runbot_branch.py

```diff
@@ -41,13 +41,10 @@ def _ssh_keyscan(self, ssh):
         The question is like to:
             'Are you sure you want to continue connecting (yes/no)?'"""
         cmd = ['ssh-keyscan', '-p']
-        match = re.search(r'@(?P<port_host>[^/]+)', ssh)
+        match = re.search(r'@([^/]+(?=:|/]))?(?::)?([^/]+)?/', ssh)
```

Comment:
@moylop260 
Better news! 
I found a regular expression to avoid a split function and another if statements